doc/getting-started.rst

```diff
@@ -1,11 +1,12 @@
-Getting Started with Haskell and Cabal
+Getting Started
 ======================================
 
-Installing the Haskell toolchain
+Installing Cabal
```

Comment:
This is the only line I am a bit unsure about: it is true that `Installing Cabal` is simpler and more beginner friendly, but maybe the beginner is reassured by something like `Installing Cabal (and the Haskell toolchain)`.

Reply:
I would like the title to focus on Cabal (since it's the Cabal documentation). The Haskell toolchain is one method; We could mention the usual OS package managers (e.g. for Debian, Ubuntu, Mac, Windows, NixOS?) or how to install Cabal from source.